54 beta indicator alt

tessaSAC opened the pull request:
[Like 54 beta pr](https://github.com/AngularNYC/angular-presentation/pull/117) except it
- increases size of ripples in animation
- moves feedback request slide to the index component
- adds functionality to move to next slide onclick/onkeypress

Known issues:
- Ripple looks funny on overview mode and there's a random `1` in the bottom right corner
- Would love to hide `next` arrow
- Right now it seems like it will always show on the first load rather than the first load for that user -- don't know if this is ok or not

kirjs commented:
I have an idea how this could be done without SCSS and with less code, let's chat in-person :)

tessaSAC commented:
Added the changes Kirill showed me at AskJS earlier.

However something got corrupted in my local copy (looked like two branches combined somehow) so I re-pulled the repo after which this branch would no longer work for me?? (Weird because I saw Kirill switch to it earlier with no problems.)

I combined with master which resulted in the return of the routing error message and the feedback-slide component placement was completely off-center but I couldn't find anything in the git diff.

Finally I noticed something looked off about the `no-padding` class in the Typescript component and changed the intro slide's to match -- was that the right change to make? Was the implementation of `no-padding` changed somehow?

tessaSAC commented:
Fixes #54
